python/tank/authentication/resources/login_dialog.ui

```diff
@@ -57,6 +58,21 @@ QComboBox
     margin-right: 3px;
 }
 
+QPushButton:focus
+{
+    border-color: rgb(48, 167, 227);
```

Comment:
It would be great if we can define color variables to improve UI maintainability.

Reply:
I believe QT5 only supports CSS2. Variables come in CSS3 :(